test/type/explicit_existential.swift

```diff
@@ -1,4 +1,5 @@
-// RUN: %target-typecheck-verify-swift -enable-experimental-feature NoncopyableGenerics
+// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -enable-upcoming-feature ExistentialAny -verify-additional-prefix explicit-any-
```

Comment:
I'm not sure why `swift6` version did not have it but please add one more run command with `-swift-version 6` for completeness.

Reply:
Good idea. Note that we have excluded this feature from `-swift-version 6`.